client/packages/electron/src/nativePrint.ts

```diff
@@ -0,0 +1,33 @@
+import { BrowserWindow } from 'electron';
+
+export const nativePrint = async (html: string) => {
```

Comment:
I'm curious as to why the change? the move to native printing in electron removes the print preview, so the overall user experience is worse. I'm not sure what the benefit is..?